broadway_api/daos/worker_node.py

```diff
@@ -38,9 +39,25 @@ def find_by_hostname(self, hostname):
             self._collection.find_one({WorkerNodeDao.WORKER_HOSTNAME: hostname})
         )
 
-    def find_by_liveness(self, alive):
+    def find_by_liveness(self, alive, use_ws=None):
+        pattern = {WorkerNodeDao.ALIVE: alive}
+
+        if use_ws is not None:
+            pattern[WorkerNodeDao.USE_WS] = use_ws
+
+        return list(map(self._from_store, self._collection.find(pattern)))
+
+    def kill_all_ws_nodes(self):
```

Comment:
Nit: Maybe rename this as `reset_worker_nodes`. This function is not actually killing any node but rresetting their alive property in the DB.

Reply:
Bump

Reply:
oops

Reply:
yeah that makes sense